.changeset/quiet-poems-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/reset.css": major
```

Comment:
Reset is a bit tricky to consume, once you add it, all non-explicit borders (like we had in our docs/stories) will disappear requiring intervention to bring them all back...

Not entirely sure where to convey this message 🤔

Reply:
What about something like: 

> Initialising this package with a reset to the border properties to support Tailwind's border default settings.
> - Allows for simpler use of border util classes, ie: `border-l`
> - This will also remove all non-explicit borders, ie: inheritance from native browser styles
>  - Check your chromatic snapshots and add sensible defaults where required